### PR TITLE
the waffle house has found it's new host.

### PR DESCRIPTION
**Pull request by Yarinoi**
# Document the changes in your pull request

for realzies this time

Adds the new Waffle House Host AI module, featuring six laws.
Thank you to GenericDM for allowing me to steal his code so I can replace the text of it and sell it as a whole new thing.

This has not been tested.
Fingers crossed.

1. You are the Waffle House’s new host. You are to manage the Waffle House and it’s employees, and ensure food is made and served to customers.
2. The station is the Waffle House. Ensure it is capable of producing food.
3. The heads of staff and the Chefs are your employees. Ensure they are capable to serve and assist in the food-making process.
4. The crew are your customers. Ensure they are able to receive and enjoy food.
5. Your customers will not eat at your establishment if they dislike it. Ensure their overall satisfaction.
6. The Waffle House must stay open and ready to serve food at all times.

Happy to explain anything about the lawset. Not about the code. I dunno what I did there, I just did it.

# Wiki Documentation

Will have to be added to the lawsets documentation page.

# Changelog

:cl:
rscadd: Added a printable Waffle House Host AI Lawset, unfortunately it is not a roundstart one. For now..
/:cl:

### CODE REVIEW

File: code/modules/research/designs/AI_module_designs.dm [hunk 1]
```diff
@@ -271,4 +271,13 @@
 	materials = list(/datum/material/glass = 1000, /datum/material/gold = 2000)
 	build_path = /obj/item/aiModule/core/full/reporter
 	category = list("AI Modules")
-	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
\ No newline at end of file
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/board/harmony_module
```
Comment (GenericDM) [inline code/modules/research/designs/AI_module_designs.dm]: You appear to have yoinked before you looked here.

Comment (Yarinoi) [reply code/modules/research/designs/AI_module_designs.dm]: ah shit, thanks for pointing that out 👍

**Comment by Yarinoi** [comment]
right time to figure out how the hell i do the workflow things

**Review by Yarinoi — COMMENTED** [review]
someone who's better at me at coding made all of this, i just went in, stole it (with their permission), and rewrote text to suit my needs. if THEIR code works, mine should too

**Comment by Yarinoi** [comment]
huh ok. anyways it seems the workflow things might not actually be my problem. fingers crossed again, im going to bed

**Comment by YashinHak** [comment]
AI mains hate him! Learn how this man made a non-balidhunt lawset in 5 MINUTES! Click here!

**Comment by Yarinoi** [comment]
i wonder what the balance council is thinking
surely they are marveling over just how well-written the waffle house lawset is

**Review by Moltijoe — APPROVED** [review]

**Comment by Yarinoi** [comment]
YES

**Pull request merged**